src/main/java/com/salesforce/storm/spout/sideline/config/SidelineConfig.java

```diff
@@ -51,6 +51,15 @@
     public static final String TRIGGER_CLASS = "sideline.trigger_class";
 
     /**
+     * (Integer) Interval (in seconds) to check running sidelines and refresh them if necessary.
+     */
+    @Documentation(
```

Comment:
We may want to say what the default value is here for reference.

Reply:
Done.